src/actors/upgrade/common/verify_pkg_rhsigned/verify_pkg_rhsigned.py

```diff
@@ -47,11 +45,17 @@
                 if sign in out:
                     break
             else:
-                error.append({keys['err_ctx']: context,
-                              keys['err_value']: "{} is not signed by Red Hat".format(pkg)})
-
-out = {}
-if error:
-    out.update({keys['out']: [{'value': error}]})
-
-print(json.dumps(out))
+                not_signed.append(pkg)
+
+if not_signed:
+    check_result = [{
+        'check_id': context,
```

Comment:
I'd just include the actor name here, if we need to scope the check better I'd add another field to the output structure - perhaps use something like `check_actor = <actor_name>` and `check_action = <optional_value>`. 
If we mash too much information into a single string value, we, or others, are destined to reverse-engineer the contents of the value sooner or later.